### PR TITLE
Support EventHub ConsumerGroups in Script

### PR DESCRIPTION
**Pull request by MikeStall**

### CODE REVIEW

File: src/WebJobs.Script/Description/Binding/EventHubBindingMetadata.cs [hunk 1]
```diff
@@ -11,6 +11,9 @@ public class EventHubBindingMetadata : BindingMetadata
         [AllowNameResolution]
         public string Path { get; set; }
 
+        // Optional Consume group
```
Comment (mathewc) [inline src/WebJobs.Script/Description/Binding/EventHubBindingMetadata.cs]: nit: Consumer

File: src/WebJobs.Script/Description/Binding/EventHubBindingMetadata.cs [hunk 2]
```diff
@@ -11,6 +11,9 @@ public class EventHubBindingMetadata : BindingMetadata
...
+        public string ConsumerGroup { get; set; }
```
Comment (mathewc) [inline src/WebJobs.Script/Description/Binding/EventHubBindingMetadata.cs]: the json ToObject<EventHubBinding>() we have in place is what makes this work, right? If they use the right name in their binding metadata, this gets deserialized?

Comment (MikeStall) [inline src/WebJobs.Script/Description/Binding/EventHubBindingMetadata.cs]: Yes & Yes.

**Comment by mathewc** [comment]
:ship:

**Pull request closed without merging**